File: app/src/main/java/com/jann_luellmann/thekenapp/view/DrinkHeading.java
package com.jann_luellmann.thekenapp.view;

import android.content.Context;
import android.widget.LinearLayout;
import android.widget.TableRow;
import android.widget.TextView;

import com.jann_luellmann.thekenapp.R;
import com.jann_luellmann.thekenapp.data.model.Drink;

import java.util.List;

public class DrinkHeading extends TableRow {

    private LinearLayout linearLayout;

    public DrinkHeading(Context context, List<Drink> drinks) {
        super(context);
        init();

        for (Drink drink : drinks) {
            TextView tv = new TextView(getContext());
            tv.setText(drink.getName() + "\n" + drink.getPrice());
            linearLayout.addView(tv);
        }
    }

    private void init() {
        inflate(getContext(), R.layout.tablerow, this);
        linearLayout = findViewById(R.id.drinks_list);
    }
}

File: app/src/main/java/com/jann_luellmann/thekenapp/view/DrinkHeading.kt
package com.jann_luellmann.thekenapp.view

import android.content.Context
import android.widget.LinearLayout
import android.widget.TableRow
import android.widget.TextView
import com.jann_luellmann.thekenapp.R
import com.jann_luellmann.thekenapp.data.model.Drink

class DrinkHeading(context: Context?, drinks: List<Drink>) : TableRow(context) {
    private var linearLayout: LinearLayout? = null
    private fun init() {
        inflate(context, R.layout.tablerow, this)
        linearLayout = findViewById(R.id.drinks_list)
    }

    init {
        init()
        for (drink in drinks) {
            val tv = TextView(getContext())
            tv.text = "${drink.name.toString()}\n${drink.price}"
            linearLayout!!.addView(tv)
        }
    }
}